lib/internal/bootstrap/pre_execution.js

```diff
@@ -237,9 +237,15 @@ function setupInspectorHooks() {
 // this is used to deprecate APIs implemented in C++ where the deprecation
 // utitlities are not easily accessible.
 function initializeDeprecations() {
-  const { deprecate } = require('internal/util');
+  const { deprecate, getSystemErrorName } = require('internal/util');
   const pendingDeprecation = getOptionValue('--pending-deprecation');
 
+  process.binding('uv').errname = deprecate(
```

Comment:
```suggestion
internalBinding('uv').errname = deprecate(
```

If we call `process.binding` the deprecation message is triggered right away and the test will fail.